### PR TITLE
Renamed duplicated class in somatic_wgs_sv_calling

### LINKED ISSUE #41: Class defined twice in file
**Bug description:** 
Class `SomaticWgsSvCallingWorkflow` defined twice in `snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py`.
Rename or remove one of the classes.

**Additional info:**
Codacy issue: https://app.codacy.com/gh/bihealth/snappy-pipeline/file/54235663449/issues/source?bid=22498844&fileBranchId=22498844#l378

### PR DESCRIPTION
**Pull request by eudesbarbosa**
closes #41

### CODE REVIEW

**Comment by holtgrewe** [comment]
Ist this still a draft?

**Comment by eudesbarbosa** [comment]
No, ready for review.

File: snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py [hunk 1]
```diff
@@ -80,9 +80,10 @@
 from biomedsheets.shortcuts import CancerCaseSheet, CancerCaseSheetOptions, is_not_background
 from snakemake.io import expand
 
-from ..abstract import BaseStepPart, BaseStep, LinkOutStepPart
-from ..ngs_mapping import NgsMappingWorkflow
-from ...utils import listify, dictify
+from snappy_pipeline.workflows.abstract import BaseStepPart, BaseStep, LinkOutStepPart
```
Comment (holtgrewe) [inline snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py:83]: So you see this as a stylistic change? I generally use relative imports to express that the code is from the same library. Is there any drawback? I think import order enforcement or auto adjustment is still on our todo list.

Sorry this is a very general issue that I'm opening here but maybe we can make some decision here adjust accordingly and create a ticket with this for tool support and then add this laters this should also be added to some group wide guideline.

**Review by holtgrewe — CHANGES_REQUESTED** [review]

File: snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py (hunk 1 above)
Comment (holtgrewe) [reply snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py:83]: s/^So/Do/

Comment (eudesbarbosa) [reply snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py:83]: This is a stylistic change. If you look around the code you can find both options. For example, in `workflows` we generally have `...`; while in `test_*` we have `snappy_pipeline.workflows.abstract` or similar...

I'm assuming we are solving the issue by adopting Codacy standards/style.

Comment (holtgrewe) [reply snappy_pipeline/workflows/somatic_wgs_sv_calling/__init__.py:83]: In the tests, imports have to be absolute as the tests are separate from the implementation. I suggest we move everything to absolute imports and enforce import order through Codacy/CI linter. I've added #52 to adress this. I think we're good to merge here.

**Review by holtgrewe — APPROVED** [review]

**Pull request merged**